frontend/src/components/custom-tools/prompt-card/PromptCard.jsx

```diff
@@ -476,11 +509,20 @@ function PromptCard({
   };
 
   const handleOutputApiRequest = async (isOutput) => {
-    let profileManager = selectedLlmProfileId;
-    if (singlePassExtractMode) {
-      profileManager = defaultLlmProfile;
+    let url;
+    if (isSimplePromptStudio) {
+      url = promptOutputApiSps(
+        details?.tool_id,
+        promptDetails?.prompt_id,
+        null
+      );
+    } else {
+      let profileManager = selectedLlmProfileId;
+      if (singlePassExtractMode) {
+        profileManager = defaultLlmProfile;
+      }
+      url = `/api/v1/unstract/${sessionDetails?.orgId}/prompt-studio/prompt-output/?tool_id=${details?.tool_id}&prompt_id=${promptDetails?.prompt_id}&profile_manager=${profileManager}&is_single_pass_extract=${singlePassExtractMode}`;
```

Comment:
@tahierhussain Should we add something similar method here ? Feel free to resolve if not possible

Reply:
@athul-rs This API is being called only in one place in this component. However, it is used in other components as well. So, I think it would be better to address this issue in [this ticket](https://zipstack.atlassian.net/browse/UN-1435).